.github/workflows/cfn-publish-resource.yaml

```diff
@@ -36,12 +36,17 @@ jobs:
       BRANCH_NAME: ${{ github.event.inputs.branchName }}
 
       # Atlas Profile
-      ATLAS_PUBLIC_KEY: ${{ secrets.ATLAS_PUBLIC_KEY }}
-      ATLAS_PRIVATE_KEY: ${{ secrets.ATLAS_PRIVATE_KEY }}
-      ATLAS_ORG_ID: ${{ secrets.ATLAS_ORG_ID }}
+      ATLAS_PUBLIC_KEY: ${{ secrets.ATLAS_PUBLIC_KEY_PUBLISHING }}
+      ATLAS_PRIVATE_KEY: ${{ secrets.ATLAS_PRIVATE_KEY_PUBLISHING }}
+      ATLAS_ORG_ID: ${{ secrets.ATLAS_ORG_ID_PUBLISHING }}
```

Comment:
To understand, are these credentials associated to a non-dev account? Do we use this to run contract tests?

Reply:
no this are the ones that we were using to publish the resource, not for contract testing

Reply:
yeas this ones were the original ones we used them in the publishing process